src/Mimi.jl

```diff
@@ -14,50 +14,32 @@ using StringBuilders
 
 export
     @defcomp,
-    @modelegate,
     MarginalModel,
     Model,
-    addcomponent,
-    add_connector_comps,
-    add_dimension!,
-    addparameter,
-    compdef,
-    compdefs,
-    compinstance,
-    compkeys,
+    addcomponent,  # push!, add_component!, add_component, addcomp!, add!; could also have multiple functions like insert!, push! etc. instead of before after keywords
+    add_connector_comps, #should this be deleted?  Docs indicate we need it ...
     components,
-    connect_parameter,
+    connect_parameter, # connect!, bind, assign, link; ! yes or no; frontrunner: connect!
     create_marginal_model,
-    datatype,
-    description,
-    dimension,
-    dimensions,
-    disconnect!,
+    disconnect!, # disconnect_parameter!; frontrunner: disconnect!
     explore,
     getdataframe,
-    getproperty,
     gettime,
     get_parameter_value,
     get_variable_value,
     hasvalue,
-    interpolate,
     is_first,
     is_last,
     load_comps,
     modeldef,
     name,
     new_component,
     parameters,
-    # plot,
-    # plot_comp_graph,
-    replace_component,
```

Comment:
`replace_component ` -->  `replace_comp` to match `component` --> `comp`?

Reply:
Agreed.